Image/image.css

```diff
@@ -0,0 +1,49 @@
+@import '../style.css';
+
+.responsive-img-container{
+    display: flex;
+    flex-wrap: wrap;
+    width:100%;
+    height: max-content;
+    justify-content: space-evenly;
+    align-items: baseline;
+    margin:10px;
+}
+
+.img-sm{
+    height:10rem;
+    width:10rem;
+}
+
+.img-md{
+    height:20rem;
+    width:20rem;
+}
+
+.img-lg{
+    height:30rem;
+    width:30rem;
+}
+
+.img-border-border-radius{
+    border:2px solid var(--lightgray-color);
+    border-radius:2rem;
+    -moz-border-radius:2rem;
+    -webkit-border-radius:2rem;
+}
+
+.img-shadow{    
+    -moz-box-shadow: 10px 10px 5px var(--lightgray-color);
+    -webkit-box-shadow: 10px 10px 5px var(--lightgray-color);
+    box-shadow: 10px 10px 5px var(--lightgray-color);    
+}
+
+.img-round{
+    border:2px solid var(--shadow-color);
+    -moz-box-shadow: 0px 6px 5px var(--shadow-color);
+    -webkit-box-shadow: 0px 6px 5px var(--shadow-color);
```

Comment:
Why these 42 and 43 line of code, is it required?? 
And if it is required please put a comment explaining why it is required so it would be understood to the reviewer as this code is used quiet rarely😀

Reply:
removed 42 43 line